src/Nest/_Generated/_Requests.generated.cs

```diff
@@ -5438,12 +5466,10 @@ public partial class SegmentsRequest : PlainRequestBase<SegmentsRequestParameter
 		public bool? AllowNoIndices { get => Q<bool?>("allow_no_indices"); set => Q("allow_no_indices", value); }
 		///<summary>Whether to expand wildcard expression to concrete indices that are open, closed or both.</summary>
 		public ExpandWildcards? ExpandWildcards { get => Q<ExpandWildcards?>("expand_wildcards"); set => Q("expand_wildcards", value); }
-		///<summary>TODO: ?</summary>
-		public string OperationThreading { get => Q<string>("operation_threading"); set => Q("operation_threading", value); }
```

Comment:
Needs to be patched back in, in order to not break binary compatibility

Reply:
:+1: